lua_boxed_numerics.cpp: warning: no previous declaration for 'int lua_new_uijnt32_t(..."

While compiling for CubeBlack I'm seeing this warning

```
../../libraries/AP_Scripting/lua_boxed_numerics.cpp: In function 'int lua_new_uint32_t(lua_State*)':
../../libraries/AP_Scripting/lua_boxed_numerics.cpp:48:5: warning: no previous declaration for 'int lua_new_uint32_t(lua_State*)' [-Wmissing-declarations]
 int lua_new_uint32_t(lua_State *L) {
     ^~~~~~~~~~~~~~~~
At global scope:
cc1plus.exe: warning: unrecognized command line option '-Wno-expansion-to-defined'
```

--- a/libraries/AP_Scripting/lua_boxed_numerics.cpp
+++ b/libraries/AP_Scripting/lua_boxed_numerics.cpp
@@ -45,6 +45,7 @@ int new_uint32_t(lua_State *L) {
 }
 
 // the exposed constructor to lua calls to create a uint32_t
+int lua_new_uint32_t(lua_State *L);
 int lua_new_uint32_t(lua_State *L) {
     const int args = lua_gettop(L);
     if (args > 1) {

